components/expression/__examples__/example.jsx

```diff
@@ -0,0 +1,498 @@
+									log({
+										action: this.props.action,
+										event,
+										eventName: `Formula updated in Text Editor`,
+										data,
+									}),
+								onClickCheckSyntax: () =>
+									log({
+										action: this.props.action,
+										event,
+										eventName: `Check Syntax Button Clicked`,
+										data: null,
+									}),
+								onClickHelp: () =>
+									log({
+										action: this.props.action,
+										event,
+										eventName: `Get Help Button Clicked`,
+										data: null,
+									}),
+							}}
+							functionCombobox={
+								<Combobox
+									labels={{
+										placeholder: 'Insert a Function',
+									}}
+									id="expression-formula-insert-function"
+									options={ResourcesList}
+									selection={
+										this.state.conditions.function
+											? [
+													find(ResourcesList, {
+														id: this.state.conditions.function,
+													}),
+												]
+											: []
+									}
+									events={{
+										onSelect: (event, data) => {
+											this.updateFormula(data.selection[0].id, 'function');
+											log({
+												action: this.props.action,
+												event,
+												eventName: `Formula Function Changed`,
+												data,
+											});
+										},
+									}}
+									variant="inline-listbox"
+								/>
+							}
+							operatorInput={
+								<Input
+									assistiveText={{ label: 'Insert a Operator' }}
+									id="insert-operator-formula"
+									placeholder="Insert a Operator"
+								/>
+							}
+						/>
+					) : (
+						this.state.conditions.map(
+							(condition, i) =>
+								!condition.isGroup ? (
+									<ExpressionCondition
+										/* eslint-disable-next-line react/no-array-index-key */
+										key={i}
+										id={`expression-condition-${i}`}
+										labels={{
+											label: Example.getTriggerType(i, this.state.triggerType),
+										}}
+										events={{
+											onChangeOperator: (event, obj) => {
+												log({
+													action: this.props.action,
+													event,
+													eventName: `Condition ${i} Operator Changed`,
+													data: obj,
+												});
+												this.updateData(i, obj, 'operator');
+											},
+											onChangeResource: (event, obj) => {
+												log({
+													action: this.props.action,
+													event,
+													eventName: `Condition ${i} Resource Changed`,
+													data: obj,
+												});
+												this.updateData(i, obj, 'resource');
+											},
+											onChangeValue: (event, data) => {
+												log({
+													action: this.props.action,
+													event,
+													eventName: `Condition ${i} Value Changed`,
+													data,
+												});
+												this.updateData(i, data.value, 'value');
+											},
+											onDelete: () => {
+												log({
+													action: this.props.action,
+													event,
```

Comment:
Same here, since there's no event variable here, set this to either an empty object or `null`